moz_kinto_publisher/main.py

```diff
@@ -961,6 +961,136 @@ def publish_crlite(*, args, ro_client, rw_client):
             )
 
 
+def publish_ctlogs(*, args, ro_client, rw_client):
+    # Copy CT log metadata from google's v3 log_list to our kinto collection.
+    # This will notify reviewers who can then manually enroll the log in CRLite.
+    #
+    # Schema for our ct-logs kinto collection:
+    #   {
+    #       "crlite_enrolled": boolean,
+    #       "description": string,
+    #       "key": string,
+    #       "log_id": string,
+    #       "mmd": integer,
+    #       "url": string
+    #   }
+    #
+
+    log_list_json = requests.get(
+        "https://www.gstatic.com/ct/log_list/v3/log_list.json"
+    ).json()
+
+    # Google groups CT logs according to their operators, we want a flat list
+    upstream_logs_raw = [
+        ctlog for operator in log_list_json["operators"] for ctlog in operator["logs"]
+    ]
+
+    # Translate |upstream_logs_raw| to our schema (and remove unused fields)
+    upstream_logs = [
+        {
+            "crlite_enrolled": False,
+            "description": ctlog["description"],
+            "key": ctlog["key"],
+            "logID": ctlog["log_id"],
```

Comment:
Either I'm confused or `log_id` in the comment about our schema should be `logID`.

Reply:
The comment was wrong. Thanks.